tools/check-templates

```diff
@@ -1,13 +1,16 @@
 #!/usr/bin/env python3
 import argparse
 import logging
+import os
 import sys
 
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), ".."))
+
 # check for the venv
-from lib import sanity_check
-from lib.html_branches import build_id_dict
-from lib.pretty_print import validate_indent_html
-from lib.template_parser import validate
+from tools.lib import sanity_check
+from tools.lib.html_branches import build_id_dict
+from tools.lib.pretty_print import validate_indent_html
+from tools.lib.template_parser import validate
```

Comment:
I think all the non-`sanity_check` imports might belong after the `check_venv` logic, since they may import things like `zulint` that come from that venv.

Reply:
I think this is the only instance of this issue; I'll just adjust and merge.